### PR TITLE
feat(page): Complete Your User Profile - No Associated Financial Institutions with Email Domain - Phase 2

### PR DESCRIPTION
**Pull request by shindigira**
Phase 2 of #220

## Changes
- feat(form submission): Adds the `Your request has been submitted` scenario. Reached by after a non-email domain submission on the Complete Your User Profile
- feat(for submission): Refines the errors for instances of multiple financial institutions manually added
- chore(RSSD ID): Removed RSSD ID from the Complete Your User Profile - Mail API
- feat(Mail API): Updated to use the double api request to function with the Mail API changes

## How to Test
**Test 1**
- Navigate to `/profile/complete` as a user with an email domain **without** associated financial institutions
- Hit `Add financial institutions` a few times and do _not_ fill out the fields
- Hit `Submit` and verify the messages in the error header as shown in the screenshot

**Test 2**
- Navigate to `/profile/complete` as a user with an email domain **with** associated financial institutions
- Successfully fill out the form and hit 'Submit'
- Verify the summary as shown in the screenshot

## Screenshots

|Errors Header|Summary|Removed RSSD ID|
|---|---|---|
|Screenshot 2024-02-21 at 3 54 52 PM|Screenshot 2024-02-21 at 3 55 17 PM|Screenshot 2024-02-26 at 4 27 29 PM|

### CODE REVIEW

**Review by meissadia — COMMENTED** [review]
Test 1 - Error headers work great! 👍🏾 

Test 2 - Fails for me.  I've updated all repos, restarted everything, still fails. 
Axios 500
    Screenshot 2024-02-21 at 4 55 33 PM

**Comment by billhimmelsbach** [comment]
> Test 2 - Fails for me. I've updated all repos, restarted everything, still fails.

Here's the line for you to add to the `.env` file @meissadia 
https://github.com/cfpb/sbl-frontend/blob/main/ENV-GUIDE.md?plain=1#L13

**Comment by meissadia** [comment]
So I've checked out Jason's branch with this fix and I'm able to get the Mail API running. 👍🏾 
 
Unfortunately I'm still not able to submit data for `Test 2`. I no longer see the error overlay but get a `500 Internal Server Error` in the devtools.

File: src/pages/ProfileForm/ProfileFormUtils.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ export const scrollToElement = (name = 'FormErrorHeader'): void => {
   scroller.scrollTo(name, {
     duration: 375,
     smooth: true,
-    offset: -35, // Scrolls to element 35 pixels above the element
+    offset: -25, // Scrolls to element 35 pixels above the element
```
Comment (meissadia) [inline src/pages/ProfileForm/ProfileFormUtils.ts:37]:
```suggestion
    offset: -25, // Scrolls to element with additional top space
```

**Review by meissadia — APPROVED** [review]
Looks great! Latest update fixes the Mail issue for me, thanks for digging in there.

**Pull request merged**